optuna/study/study.py

```diff
@@ -308,6 +285,42 @@ def _get_trials(
 
         return self._storage.get_all_trials(self._study_id, deepcopy=deepcopy, states=states)
 
+    def _get_best_trial(self, deepcopy: bool = True) -> FrozenTrial:
```

Comment:
How about making the argument mandatory? All the lines using this function specify the argument, and the Optuna developers should be aware of the copy.
```suggestion
    def _get_best_trial(self, deepcopy: bool) -> FrozenTrial:
```